organizationinvitation/client.go

```diff
@@ -47,3 +48,56 @@ func (c *Client) Create(ctx context.Context, params *CreateParams) (*clerk.Organ
 	err = c.Backend.Call(ctx, req, invitation)
 	return invitation, err
 }
+
+type ListParams struct {
+	clerk.APIParams
+	clerk.ListParams
+	Statuses *[]string
+}
+
+func (p *ListParams) ToQuery() url.Values {
+	q := p.ListParams.ToQuery()
+
+	if p.Statuses != nil && len(*p.Statuses) > 0 {
+		q["status"] = *p.Statuses
+	}
+
+	return q
+}
+
+// List returns a list of organization invitations
+func (c *Client) List(ctx context.Context, organizationID string, params *ListParams) (*clerk.OrganizationInvitationList, error) {
```

Comment:
:wrench: Let's group all parameters in a single type, similar to https://github.com/clerk/clerk-sdk-go/blob/000038eca0d2542ae5c318a332c82dc2514eaeb4/organizationmembership/client.go#L131